obspy/clients/fdsn/tests/test_client.py

```diff
@@ -1143,6 +1144,13 @@ def test_redirection_auth(self):
         # Just make sure something is being downloaded.
         self.assertTrue(bool(len(inv.networks)))
 
+    def test_no_data(self):
+        """
+        Verify that a request returning no data raises an identifiable exception
+        """
+        self.assertRaises(FDSNNoDataException, self.client.get_events,
+            starttime=UTCDateTime("2001-01-07T01:00:00"),
+            endtime=UTCDateTime("2001-01-07T01:01:00"), minmagnitude=8)
```

Comment:
The CI failures are code formatting issues. Please make sure the indentation is according to our rules. Just running `flake8` over the changed files enables you to judge it locally.